src/clarity-angular/tabs/vtabs-content.ts

```diff
@@ -0,0 +1,42 @@
+import {Component} from "@angular/core";
+
+
+/**
+ * @class VTabContent
+ *
+ * @description
+ * A child component of VTabs that:
+ * 1. Provides an id property set and used by the VTabs class to map VTabsNavLink items to the corresponding
+ *    VTabsContent items
+ * 2. Provides an activated property that triggers Angular to add/remove the content to the DOM via *ngIf
+ */
+@Component({
+    selector: "clr-vtabs-content",
+    template: `
+        <section class="vtabs-content" *ngIf="activated" role="tabpanel">
```

Comment:
Can we not add the `vtabs-content` class and the `role` on the host element and replace this with `ng-container` with just the `*ngIf`? The angular component otherwise will be even more nested no? We now have 
clr-vtabs-content > .vtabs-content > ng-content whereas the static version has just the last 2.